### DISCUSSION TITLE
Update telegram page

### DISCUSSION THREAD

**Discussion by minecraftchest1** [42wim/matterbridge#1365] [General]
I updated [https://github.com/42wim/matterbridge/wiki/Section-Telegram-(basic]( https://github.com/42wim/matterbridge/wiki/Section-Telegram-(basic))) with some info I discovered today,

**Comment by qaisjp** [comment]
Nice, and thank you! I've tweaked the formatting a little bit :)

**Reply by minecraftchest1** [reply]
I can accept the changes.

**Reply by SonaliBendre** [reply]
possible to use a dedicated tg user(user-bot) instead of bot? As telegram bot can't interact with other bots, so we can't use telegram bots from slack,discord,irc etc. Some telegram bots are awesome which I've not seen in other platforms, so i would want a user-bot to bridge telegram to use these tg bots. For this we'll need (Go Mtproto Lib?)

**Reply by minecraftchest1** [reply]
From what I have seen, it is not possible with Matterbridge currently.
